packages/components/src/RichTextEditor/RichTextEditor/_docs/RichTextEditor.mdx

```diff
@@ -0,0 +1,51 @@
+import { Canvas, Controls, DocsStory, Meta } from "@storybook/blocks"
+import { ResourceLinks, KaioNotification, Installation } from "~storybook/components"
+import * as RichTextEditorStories from "./RichTextEditor.stories"
+
+<Meta of={RichTextEditorStories} />
+
+# RichTextEditor
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/RichTextEditor/RichTextEditor"
+  figma="https://www.figma.com/file/ZRfnoNUXbGZv4eVWLbF4Az/%EF%B8%8F%F0%9F%96%BC%EF%B8%8F-Component-Gallery?node-id=9%3A86406&t=P1w10jr2cpPuaayw-1"
+  designGuidelines="https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3081896752/Rich+Text+Editor"
+  className="!mb-8"
+/>
+
+<KaioNotification />
+
+<Installation
+  installCommand="yarn add @kaizen/components"
+  importStatement='import { RichTextEditor } from "@kaizen/components"'
+/>
+
+## Overview
+
+A text area with additional capabilities for users to format the input text.
+
+<Canvas of={RichTextEditorStories.Playground} />
+<Controls of={RichTextEditorStories.Playground} />
+
+## API
+
+<DocsStory of={RichTextEditorStories.Controls} />
```

Comment:
Would be good to circle back later and add more to this in terms of current controls available and how to handle grouping